### PR TITLE
DTFS2-4412

### PR DESCRIPTION
**Pull request by eddieleeper**
Remove api calls.

### CODE REVIEW

**Review by ttbarnes — DISMISSED** [review]

File: portal-api/src/v1/gef/controllers/application-submit.js [hunk 1]
```diff
@@ -28,7 +28,12 @@ const generateUkefDealId = async (application) => {
   };
 };
 
-const generateUkefFacilityId = (facility) => (facility.ukefFacilityId !== null ? facility.ukefFacilityId : generateId('facility'));
+const generateUkefFacilityId = async (facility) => {
+  const ukefFacilityId = facility.ukefFacilityId !== null ? facility.ukefFacilityId : await generateId('facility');
```
Comment (davehlewis) [inline portal-api/src/v1/gef/controllers/application-submit.js:32]: Could this not be written as `facility.ukeffacilityId || await generateId('facility')` or is there a chance of a legitimate falsey value?

Comment (eddieleeper) [reply portal-api/src/v1/gef/controllers/application-submit.js:32]: Thanks @davehlewis - Yes, it probably could - when testing like that I got a failure, so added the explicit check.  I'll have a look at refactoring on the next PR.  This branch is a work in progress for the various steps on DTFS-4412

Comment (davehlewis) [reply portal-api/src/v1/gef/controllers/application-submit.js:32]: No problem, can sometimes be a bit odd. If we could amend our eslint I would say nullish coalescing would work move cleanly for it.

Comment (eddieleeper) [reply portal-api/src/v1/gef/controllers/application-submit.js]: Updated.

**Review by ttbarnes — APPROVED** [review]

File: portal-api/src/v1/gef/controllers/application-submit.js [hunk 2]
```diff
@@ -52,7 +50,7 @@ const addUkefFacilityIdToFacilities = async (applicationId) => {
   const facilities = await getAllFacilitiesByApplicationId(applicationId);
 
   facilities.forEach(async (facility) => {
-    const ukefFacilityId = await generateUkefFacilityId(facility);
+    const ukefFacilityId = generateUkefFacilityId(facility);
```
Comment (davehlewis) [inline portal-api/src/v1/gef/controllers/application-submit.js:53]: why has the async been removed from these?

**Pull request merged**